Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -186,10 +189,23 @@ void GameCubePane::OnConfigPressed(int slot)
   {
     bool ok;
     const auto new_mac = QInputDialog::getText(
-        this, tr("Broadband Adapter MAC address"), tr("Enter new Broadband Adapter MAC address:"),
-        QLineEdit::Normal, QString::fromStdString(SConfig::GetInstance().m_bba_mac), &ok);
+        this, tr("Broadband Adapter (TAP) MAC address"),
+        tr("Enter new Broadband Adapter (TAP) MAC address:"), QLineEdit::Normal,
+        QString::fromStdString(SConfig::GetInstance().m_bba_tap_mac), &ok);
     if (ok)
-      SConfig::GetInstance().m_bba_mac = new_mac.toStdString();
+      SConfig::GetInstance().m_bba_tap_mac = new_mac.toStdString();
+    return;
+  }
+  case ExpansionInterface::EXIDEVICE_ETHXLINK:
+  {
+    bool ok;
+    const auto new_dest = QInputDialog::getText(
+        this, tr("Broadband Adapter (XLink Kai) Destination Address"),
+        tr("Enter IP address of device running the XLink Kai Client. \nFor more information see"
```

Comment:
This has some whitespace at the end (both after `Client.` before the `\n` as well as at the end after the URL), do those serve any particular purpose? If not, please remove them.